docs/research/streaming_architecture.md

```diff
@@ -112,3 +112,43 @@ In the BBB configuration file, we need to specify which transcription method we
 Then we configured an additional service called `bbb-transcription-controller`. We set the websocket domain (our transcription endpoint) to the German translation. This can be done inside the repo's config yaml file. This was a bit tricky, because if you clone the official repo, change the configuration and run the `app.js` it won't work. We had to do `sudo apt install bbb-transcription-controller` and configure the files in the folder of the apt installation.
 
 On our side, we had to expose the docker port for the transcription service and add a subdomain in nginx to forward this port.
+
+### Local Agreement
+
+Local agreement is a strategy for determining the longest common prefix among n audio chunks during sequence-to-sequence speech recognition. See also the corresponding [paper](https://www.isca-archive.org/interspeech_2020/liu20s_interspeech.pdf). At the core of local agreement is the idea that if a "hypothesis" (i.e., a transcription of a sequence) is validated or reinforced by additional subsequent inputs, it is unlikely to change (or need to be changed) and can therefore be considered promising.
+
+However, this approach inherently results in a system that can only provide a definitive transcription for audio chunk n once audio chunk n+1 has been transcribed. Before the transcription of n+1, the conditions required for the transcription of n to be deemed "promising" are not yet satisfied.
+
+Our first step was to apply local agreement to partial transcriptions directly, without making further changes to the partial-to-final system already in place. This already yielded improvements over the baseline (see below). The use of local agreement stabilized partial transcriptions and eliminated jitter between them, albeit with the tradeoff of some added latency.<details>
+
+  <summary>benchmarking results</summary>
```

Comment:
the results do look very nice, but what are the latency impacts, are they significant or neglectable?

Reply:
According to the benchmarks the actual results (consider: Our streaming solution in the current state is not optimal to begin with) indicate that using local agreement in our case is even faster. 
That might also be the offset created by other processes on the host computer (benchmarks were run in WSL) but either way there does not seem to be a degraded latency

Reply:
Resolve this and merge if this answer addressed your question.